src/sentry/event_manager.py

```diff
@@ -876,28 +875,11 @@ def save(self, project_id, raw=False, assume_normalized=False):
         return event
 
     def _get_event_from_storage(self, project_id, event_id):
```

Comment:
Since this is a single statement, I would entirely revert to what we had before and just inline it.